test/services/mapping/goodsshipment/AEOMutualRecognitionPartiesBuilderSpec.scala

```diff
@@ -15,94 +15,50 @@
  */
 
 package services.mapping.goodsshipment
-import models.declaration.{DeclarationAdditionalActorsData, DeclarationAdditionalActorsDataSpec}
-import org.mockito.Mockito.when
+import forms.declaration.DeclarationAdditionalActors
+import models.declaration.DeclarationAdditionalActorsDataSpec
 import org.scalatest.mockito.MockitoSugar
 import org.scalatest.{Matchers, WordSpec}
-import play.api.libs.json.{JsArray, JsObject, JsString, JsValue}
-import uk.gov.hmrc.http.cache.client.CacheMap
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 
 class AEOMutualRecognitionPartiesBuilderSpec extends WordSpec with Matchers with MockitoSugar {
 
   "AEOMutualRecognitionPartiesBuilder " should {
```

Comment:
I think the majority of these Builder test classes have their  own set of tests for the `buildThenAdd` methods, so you can probably just delete any test that calls `SomeBuilder.build(...)`

Reply:
It varies.  Some builders have some "new" tests.  Some don't

Reply:
I imagine the ones that dont arent actually testing the `buildThenAdd`.... doh

Reply:
Trying to add missing tests as I go